### PR TITLE
DAOS-7174 doc: Update image sources from doc to docs

### PR DESCRIPTION
**Pull request by ryon-jensen**
With PR 6565 the doc folder was renamed to docs. Many image sources
and links need to be updated to point to 'docs' instead of 'doc'.

Signed-off-by: Ryon Jensen <ryon.jensen@intel.com>

Doc-only: true

Note : DAOS-7174 doesn't appear to be the correct ticket number for the doc changes, however it is what was used for PR 6565 so using it for now. Can update when correct ticket number is identified.

### CODE REVIEW

File: docs/admin/performance_tuning.md [hunk 1]
```diff
@@ -472,12 +472,12 @@ please refer to the [System Deployment: Agent Startup][6] documentation
 section.
 
 [1]: <https://github.com/daos-stack/daos/tree/master/src/cart> (Collective and RPC Transport)
-[2]: <https://github.com/daos-stack/daos/blob/master/doc/admin/installation.md#distribution-packages> (DAOS distribution packages)
-[3]: <https://github.com/daos-stack/daos/blob/master/doc/admin/installation.md#building-daos--dependencies> (DAOS build documentation)
-[4]: <https://github.com/daos-stack/daos/blob/master/doc/admin/deployment.md#server-startup> (DAOS server startup documentation)
+[2]: <https://github.com/daos-stack/daos/blob/master/docs/graphadmin/installation.md#distribution-packages> (DAOS distribution packages)
+[3]: <https://github.com/daos-stack/daos/blob/master/docs/graphadmin/installation.md#building-daos--dependencies> (DAOS build documentation)
+[4]: <https://github.com/daos-stack/daos/blob/master/docs/graphadmin/deployment.md#server-startup> (DAOS server startup documentation)
```
Comment (johannlombardi) [inline docs/admin/performance_tuning.md:477]: This should really use relative path IMO...

**Review by johannlombardi — APPROVED** [review]

**Review by cdurf1 — CHANGES_REQUESTED** [review]
All the new urls contain 'graph' (docs/graphadmin/installation.md)
Unless we are changing the names of the folders in the docs folder, graph should be removed from the address (unless specifically referencing the graph folder for image urls.

File: docs/admin/performance_tuning.md (hunk 1 above)
Comment (ryon-jensen) [reply docs/admin/performance_tuning.md]: Updated

**Comment by ryon-jensen** [comment]
> All the new urls contain 'graph' (docs/graphadmin/installation.md)
> Unless we are changing the names of the folders in the docs folder, graph should be removed from the address (unless specifically referencing the graph folder for image urls.

Oops, search and replace error. Fixed.

**Review by johannlombardi — APPROVED** [review]

**Review by cdurf1 — APPROVED** [review]

**Pull request merged**